#include "ludic_exception.hpp"

using namespace Ludic;

//////////////////////////////////////////////////////////////

Exception::Exception(String _msg) : msg(_msg) {}

//////////////////////////////////////////////////////////////

void Exception::setMessage(String msg) {
	this->msg = msg;
}

//////////////////////////////////////////////////////////////

const char* Exception::what() const throw() {
	return msg.c_str();
}

//////////////////////////////////////////////////////////////
